hoot-core/src/main/cpp/hoot/core/visitors/RemoveElementsVisitor.cpp

```diff
@@ -42,12 +42,14 @@ HOOT_FACTORY_REGISTER(ElementVisitor, RemoveElementsVisitor)
 RemoveElementsVisitor::RemoveElementsVisitor()
 {
   setConfiguration(conf());
+  _count = 0;
```

Comment:
I know it is nit-picking but throwing this in the initializer list is the "proper" thing to do.